racket/collects/racket/promise.rkt

```diff
@@ -212,3 +212,29 @@
                     (cons '#:tick       #'0.2)
                     (cons '#:use        #'0.12))])
     (λ(stx) (make-delayer stx #'delay/idle kwds))))
+
+;; A lighter-weight version of delay that doesn't care about multiple values or
+;; exceptions
+
+;; force/delay* : Delay*-Promise -> Any
+;; Like force/generic from racket/private/promise, except that it
+;; only deals with one value and doesn't care about exceptions
+;; I'm not sure whether the common code could be abstracted out or not
```

Comment:
(In reply to the first bit of https://github.com/racket/racket/pull/1182#issuecomment-168975715)
@elibarzilay I'm not sure how any common code could be abstracted out. Is this the kind of thing you meant? Or is there any way you can see to abstract anything out?